services/brig/src/Brig/API/User.hs

```diff
@@ -631,6 +636,9 @@ changeHandle uid mconn hdl allowScim = do
             || allowScim == AllowSCIMUpdates
         )
         $ throwE ChangeHandleManagedByScim
+      hasE2EId <- lift . liftSem . userUnderE2EId . userId $ u
+      when (hasE2EId && userHandle u `notElem` [Nothing, Just hdl]) $
```

Comment:
maybe?

```suggestion
      when (hasE2EId && (not $ all (== hdl) $ userHandle u)) $
```

Reply:
i like mine better :shrug: unless you insist?

Reply:
All good :)